tests/system/test_system.py

```diff
@@ -41,19 +43,21 @@
 _RESOURCE_ID = unique_resource_id("-")
 DEFAULT_FILTER = "logName:syslog AND severity>=INFO"
 DEFAULT_DESCRIPTION = "System testing"
+_TIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%f%z"
 retry_429 = RetryErrors(TooManyRequests)
 
+_ten_mins_ago = datetime.now(timezone.utc) - timedelta(minutes=10)
+_time_filter = f'timestamp>="{_ten_mins_ago.strftime(_TIME_FORMAT)}"'
 
-def _consume_entries(logger):
-    """Consume all log entries from logger iterator.
 
+def _consume_entries(logger):
+    """Consume all recent log entries from logger iterator.
     :type logger: :class:`~google.cloud.logging.logger.Logger`
     :param logger: A Logger containing entries.
-
     :rtype: list
     :returns: List of all entries consumed.
     """
-    return list(logger.list_entries())
+    return list(logger.list_entries(filter_=_time_filter))
```

Comment:
nit: this looks really ugly. Is it a python convention to name variables with underscores as both the prefix and suffix? At least, maybe worth having a space before and after "=" for clarity.

Reply:
`filter_` is used because `fitler` is a keyword (I think it's part of the code generation process). Prefix underscores are more standard for internal variables.

Unfortunately we can't add spaces because it wouldn't be accepted by the linter. I think since this is all test code it should be fine though